### DISCUSSION TITLE
Zen mode in Zed

### DISCUSSION THREAD

**Discussion by avramukk** [zed-industries/zed#20562] [Feature Requests]
Is it possible to open Zed in a completely borderless mode like nvim?
I've searched and didn't find anything like that in the settings. Could you add this to the settings?
![Zed 2024-11-12 23 43 37](https://github.com/user-attachments/assets/df277d31-679f-4cd9-87f3-0c9d66ac63c2)
![CleanShot X 2024-11-13 00 00 34](https://github.com/user-attachments/assets/9d1a84e8-6065-472b-84c5-53ba540fd02c)

**Comment by gldtn** [comment]
Not quite yet, at the moment you can't  hide title bar and status bar. See: this issue here: I believe this is similar to this request: https://github.com/zed-industries/zed/issues/4382

**Comment by SomeoneToIgnore** [comment]
https://github.com/zed-industries/zed/issues/4382
